### PR TITLE
[BPK-2221] Update RN calendar readme

### PR DESCRIPTION
**Pull request by georgegillams**

### CODE REVIEW

File: packages/react-native-bpk-component-calendar/README.md [hunk 1]
```diff
@@ -14,14 +14,15 @@ Because this package ships with native code, it is also necessary to add some na
...
+Then you'll need to add a dependency to your Podfile using the path to the NPM package as follows:
 
- - `react-native-bpk-component-calendar`, using a relative path via `node_modules`
-
-For example:
 ```
   pod 'react-native-bpk-component-calendar', path: '../node_modules/react-native-bpk-component-calendar'
 ```
+
+Note that the `react-native-bpk-component-calendar` depends on [Backpack](https://cocoapods.org/pods/Backpack). If your Podfile also depends on this directly, both must require the same version.
```
Comment (k0nserv) [inline packages/react-native-bpk-component-calendar/README.md]: They need to have a compatible semver range not be the same exact version

File: packages/react-native-bpk-component-calendar/README.md [hunk 2]
```diff
@@ -36,7 +37,7 @@ const minDate = new Date(Date.UTC(2019, 0, 2));
 To format the dates for display use
 
 ```javascript
-const locale = 'en-GB';
+const locale = 'en-gb';
```
Comment (k0nserv) [inline packages/react-native-bpk-component-calendar/README.md]: [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) codes should be capitalized by convention I believe

File: packages/react-native-bpk-component-calendar/README.md [hunk 3]
```diff
@@ -51,25 +52,31 @@ import { StyleSheet, View } from 'react-native';
...
+    this.setState({
+      selectedDates: newDates,
+    });
+  };
+
+  render() {
+    const { selectionType, onChangeSelectedDates, ...rest } = this.props;
+    return (
+      <BpkCalendar
+        locale={'en-gb'}
```
Comment (k0nserv) [inline packages/react-native-bpk-component-calendar/README.md]: `en-GB` ;)

File: packages/react-native-bpk-component-calendar/README.md [hunk 4]
```diff
@@ -51,25 +52,31 @@ import { StyleSheet, View } from 'react-native';
...
+        selectionType={SELECTION_TYPES.range}
```
Comment (k0nserv) [inline packages/react-native-bpk-component-calendar/README.md]: Can we show `minDate` and `maxDate` to drive home how to create the dates in UTC correctly?

File: packages/react-native-bpk-component-calendar/README.md (hunk 3 above)
Comment (tiagoengel) [reply packages/react-native-bpk-component-calendar/README.md]: That won't work on Android though

**Pull request merged**